### DISCUSSION TITLE
网页无法购票需要前往app

### DISCUSSION THREAD

**Discussion by NagasakiS** [mikumifa/biliTickerBuy#382] [提问专区] [answered]
![Desktop Screenshot 2024 07 06 - 00 13 39 11](https://github.com/mikumifa/biliTickerBuy/assets/174821136/7afb8237-0bbb-45b8-b339-11645b8f6194)

**Comment by MUSESTAR-Official** [comment, marked as answer]
直接复制链接到程序里就行
https://show.bilibili.com/platform/detail.html?id=85939

**Reply by NagasakiS** [reply]
就这个页面也能抢是吗，谢谢

**Reply by MUSESTAR-Official** [reply]
是的

**Reply by NagasakiS** [reply]
麻烦再问一下，抢票的时候要开着这个网页吗
